src/definitions/modules/popup.less

```diff
@@ -91,6 +91,7 @@
     position: absolute;
     content: '';
     font-size: @medium;
+    text-shadow: none;
```

Comment:
Should we put this into a variable and also add this to the usual `.ui.popup` definition? Just in case somebody wants to add a text-shadow to popup and tooltip in a theme

Reply:
And this should also be inside `:after` pseudo where the content is set. `:before` would not apply the text-shadow because it has no content and it's only to create arrow.

Reply:
Oops. 
Moved to `:after` pseuod selector.

IMHO, adding new variable for customizing `text-shadow` is unnecessary, since text-shadow in popup/tooltip are less readable.
Even without new variable, users could override text-shadow if they actually needs text-shadow.